DEV_3/DEV_3/Dev3.cs

```diff
@@ -0,0 +1,73 @@
+﻿
+namespace DEV_3
+{
+    using System;
+
+    /// <summary>
+    /// The main class.
+    /// Contains a entry point method of program.
+    /// </summary>
+    public class Dev3
+    {
+        /// <summary>
+        /// The entry point of the program
+        /// </summary>
+        /// <param name="args">Entry string.Specified in the input argument string
+        /// args[0] - customers available money
+        /// args[1] - required productivity
+        /// args[2] - criteria
+        /// </param>
+        public static void Main(string[] args)
```

Comment:
too big main method. Lets try to move some functionality to ArgumentsChecker for example, CriterioSelector or something like this